diff --git a/PKGBUILD b/PKGBUILD
index 4c5871d..1e8c7a5 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -22,7 +22,7 @@
 package() {
   cd "$_pkgname-v$pkgver"
 
-  local uuid=$(grep -Po '(?<="uuid": ")[^"]*' shell-configurator@adeswanta/metadata.json)
+  local uuid=$(grep -Po '(?<="uuid": ")[^"]*' src/metadata.json)
   install -d "$pkgdir/usr/share/gnome-shell/extensions/${uuid}"
   bsdtar -xvf ${uuid}.shell-extension.zip -C "$pkgdir/usr/share/gnome-shell/extensions/${uuid}"
 }
