diff --git a/PKGBUILD b/PKGBUILD
index 9e62ec5..7797f24 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,9 +4,9 @@ _name='Uno Calculator'
 _pkgname='uno-calculator'
 pkgname="${_pkgname}-bin"
 
-pkgver=1.2.4.725
+pkgver=1.2.5.82
 pkgrel=1
-_snap_id='ZZjSue7jjdQXHsNz5RI9BH3Pf3UnRo0V_15'
+_snap_id='ZZjSue7jjdQXHsNz5RI9BH3Pf3UnRo0V_17'
 
 pkgdesc='Uno port of Windows Calculator'
 arch=('x86_64')
@@ -23,7 +23,7 @@ noextract=("${_pkgname}-${pkgver}-x86_64.snap")
 # https://snapcraft.io/uno-calculator
 # curl -H 'Snap-Device-Series: 16' http://api.snapcraft.io/v2/snaps/info/uno-calculator | jq
 source_x86_64=("${_pkgname}-${pkgver}-x86_64.snap::${_url_source}/${_snap_id}.snap")
-sha256sums_x86_64=('dbeef8687f3fa4404bd35f57aa4c86034f89d89f71fab0152e69817f16ed5c03')
+sha256sums_x86_64=('eb2cbe9826c9e1d7302b4ebb538efa50f1ab85fcb56c5ebaed519ccd6714c7ce')
 
 prepare() {
   echo -e "#!/bin/sh\ncd /opt/${_pkgname} && exec ./Calculator.Skia.Gtk \"\$@\"" > "${_pkgname}"
