diff --git a/makepkg b/makepkg
--- a/makepkg	(revision 0c496bc92953cd1bcd4f0a3373d21597c29621fe)
+++ b/makepkg	(date 1659841426817)
@@ -225,12 +225,24 @@
 }
 
 run_pacman() {
+    echo "Entered If run_pacman"
 	local cmd cmdescape
 	if [[ $1 = -@(T|Q)*([[:alpha:]]) ]]; then
 		cmd=("$PACMAN_PATH" "$@")
+		echo "************************************************************"
+		echo "${cmd[@]}"
+		echo "Entered If Condition, params are: $*"
+		echo "************************************************************"
+
 	else
 		cmd=("$PACMAN_PATH" "${PACMAN_OPTS[@]}" "$@")
 		cmdescape="$(printf '%q ' "${cmd[@]}")"
+
+        echo "************************************************************"
+		echo "${cmd[@]}"
+		echo "Entered else Condition, params are: $*"
+		echo "************************************************************"
+
 		if (( ${#PACMAN_AUTH[@]} )); then
 			if in_array '%c' "${PACMAN_AUTH[@]}"; then
 				cmd=("${PACMAN_AUTH[@]/\%c/$cmdescape}")
@@ -252,6 +264,7 @@
 			done
 		done
 	fi
+	echo "${cmd[@]}"
 	"${cmd[@]}"
 }
