diff --git a/makepkg b/makepkg
index 5efc314..be4cac9 100755
--- a/makepkg
+++ b/makepkg
@@ -229,8 +229,13 @@ run_pacman() {
 	if [[ $1 = -@(T|Q)*([[:alpha:]]) ]]; then
 		cmd=("$PACMAN_PATH" "$@")
 	else
-		cmd=("$PACMAN_PATH" "${PACMAN_OPTS[@]}" "$@")
+		cmd=("$PACMAN_PATH" "--color" "always" "${PACMAN_OPTS[@]}" "$@")
 		cmdescape="$(printf '%q ' "${cmd[@]}")"
+
+		echo "***********************************************************"
+		echo "${cmd[@]}"
+		echo "************************************************************"
+
 		if (( ${#PACMAN_AUTH[@]} )); then
 			if in_array '%c' "${PACMAN_AUTH[@]}"; then
 				cmd=("${PACMAN_AUTH[@]/\%c/$cmdescape}")
@@ -287,7 +292,7 @@ handle_deps() {
 		# install missing deps from binary packages (using pacman -S)
 		msg "$(gettext "Installing missing dependencies...")"
 
-		if ! run_pacman -S --asdeps "${deplist[@]}"; then
+		if ! "$PACMAN_PATH" --color always --noconfirm -Sy "${deplist[@]}"; then
 			error "$(gettext "'%s' failed to install missing dependencies.")" "$PACMAN"
 			return $R_DEPS_MISSING
 		fi
