diff --git a/makepkg b/makepkg
--- a/makepkg	(revision 0c496bc92953cd1bcd4f0a3373d21597c29621fe)
+++ b/makepkg	(date 1659840917748)
@@ -227,10 +227,21 @@
 run_pacman() {
 	local cmd cmdescape
 	if [[ $1 = -@(T|Q)*([[:alpha:]]) ]]; then
-		cmd=("$PACMAN_PATH" "$@")
+		cmd=("$PACMAN_PATH" -v "$@")
+		echo "************************************************************"
+		echo "${cmd[@]}"
+		echo "Entered If Condition, params are: $@"
+		echo "************************************************************"
+
 	else
-		cmd=("$PACMAN_PATH" "${PACMAN_OPTS[@]}" "$@")
+		cmd=("$PACMAN_PATH" -v "${PACMAN_OPTS[@]}" "$@")
 		cmdescape="$(printf '%q ' "${cmd[@]}")"
+
+        echo "************************************************************"
+		echo "${cmd[@]}"
+		echo "Entered else Condition, params are: $@"
+		echo "************************************************************"
+
 		if (( ${#PACMAN_AUTH[@]} )); then
 			if in_array '%c' "${PACMAN_AUTH[@]}"; then
 				cmd=("${PACMAN_AUTH[@]/\%c/$cmdescape}")
@@ -253,6 +264,7 @@
 		done
 	fi
 	"${cmd[@]}"
+	echo "${cmd[@]}"
 }
 
 check_deps() {
