diff --git a/makepkg b/makepkg
index 5efc314..8654613 100755
--- a/makepkg
+++ b/makepkg
@@ -227,10 +227,19 @@ missing_source_file() {
 run_pacman() {
 	local cmd cmdescape
 	if [[ $1 = -@(T|Q)*([[:alpha:]]) ]]; then
-		cmd=("$PACMAN_PATH" "$@")
+		cmd=("$PACMAN_PATH" -v "-y" "--color" "always" "$@")
+		echo "************************************************************"
+		echo "${cmd[@]}"
+		echo "************************************************************"
+
 	else
-		cmd=("$PACMAN_PATH" "${PACMAN_OPTS[@]}" "$@")
+		cmd=("$PACMAN_PATH" -v "-y" "--color" "always" "${PACMAN_OPTS[@]}" "$@")
 		cmdescape="$(printf '%q ' "${cmd[@]}")"
+
+		echo "***********************************************************"
+		echo "${cmd[@]}"
+		echo "************************************************************"
+
 		if (( ${#PACMAN_AUTH[@]} )); then
 			if in_array '%c' "${PACMAN_AUTH[@]}"; then
 				cmd=("${PACMAN_AUTH[@]/\%c/$cmdescape}")
