diff --git a/PKGBUILD b/PKGBUILD
index e09c6f7..faf3240 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,5 +3,5 @@
 pkgname=fatcat
-pkgver=1.1.0
+pkgver=1.1.1
 pkgrel=1
 pkgdesc=" FAT filesystems explore, extract, repair, and forensic tool"
 arch=('i686' 'x86_64')
@@ -10,7 +10,7 @@
 depends=()
 makedepends=('cmake')
 source=(${pkgname}-${pkgver}.tar.gz::"https://github.com/Gregwar/fatcat/archive/v${pkgver}.tar.gz")
-sha256sums=('303efe2aa73cbfe6fbc5d8af346d0f2c70b3f996fc891e8859213a58b95ad88c')
+sha256sums=('5574f715f62a9feda7fffc8099f8bca7134ff4f1274bb1bc0db96a2969eea206')
 
 build() {
   cd ${pkgname}-${pkgver}
@@ -29,10 +29,4 @@ package() {
   # install man page
   install -Dm644 man/fatcat.1 "${pkgdir}/usr/share/man/man1/fatcat.1"
   gzip -1 ${pkgdir}/usr/share/man/man1/fatcat.1
-  # install license
-  install -Dm644 LICENSE "${pkgdir}/usr/share/licenses/fatcat/LICENSE"
-  # copy source code
-  cp -r src ${pkgdir}/usr/share/fatcat/src
-  # copy other docs
-  cp -r docs ${pkgdir}/usr/share/fatcat/docs
 }
