diff --git a/PKGBUILD b/PKGBUILD
index e09c6f7..faf3240 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -29,9 +29,4 @@ package() {
   # install man page
   install -Dm644 man/fatcat.1 "${pkgdir}/usr/share/man/man1/fatcat.1"
+  gzip -1 ${pkgdir}/usr/share/man/man1/fatcat.1
-  # install license
-  install -Dm644 LICENSE "${pkgdir}/usr/share/licenses/fatcat/LICENSE"
-  # copy source code
-  cp -r src ${pkgdir}/usr/share/fatcat/src
-  # copy other docs
-  cp -r docs ${pkgdir}/usr/share/fatcat/docs
 }
