diff --git a/PKGBUILD b/PKGBUILD
index 5a6ba8a..dd9cd92 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -6,10 +6,9 @@
-_pyver=3.9
 pkgdesc="A tool for recovering passwords from pixelized screenshots"
 arch=('any')
 url="https://github.com/beurtschipper/Depix"
 license=('CC-BY-4.0')
-depends=('python')
-makedepends=('git')
+depends=('python-pillow')
+makedepends=('git' 'python-build' 'python-installer' 'python-setuptools' 'python-pip' 'python-wheel')
 provides=("depix")
 conflicts=("depix")
 source=("${pkgname}::git+https://github.com/beurtschipper/Depix.git")
